execute/transport.go

```diff
@@ -31,18 +37,45 @@ type consecutiveTransport struct {
 	inflight       int32
 }
 
-func newConsecutiveTransport(dispatcher Dispatcher, t Transformation) *consecutiveTransport {
+func newConsecutiveTransport(dispatcher Dispatcher, t Transformation, n plan.Node) *consecutiveTransport {
 	return &consecutiveTransport{
 		dispatcher: dispatcher,
 		t:          t,
 		// TODO(nathanielc): Have planner specify message queue initial buffer size.
 		messages: newMessageQueue(64),
+		stack:    n.CallStack(),
 		finished: make(chan struct{}),
 	}
 }
 
+func (t *consecutiveTransport) sourceInfo() string {
+	if len(t.stack) == 0 {
+		return ""
+	}
+
+	// Learn the filename at the top of the stack.
+	// We want the entry that is at the deepest depth
+	// from the primary file.
+	// The top of the call stack is the current file
+	// we are running from. This name can vary, so
+	// we retrieve it from the call stack.
```

Comment:
I'll clarify the comment since "top" is likely used incorrectly. The comment refers to top as the first entry, but nobody refers to a call stack that way. So this is using the bottom of the call stack (the main function) as its reference.